lochness/utils/path_checker.py

```diff
@@ -235,6 +235,32 @@ def update_by_removing_unused_files(df: pd.DataFrame) -> None:
     ds_store_index = df[df.file_name == '.DS_Store'].index
     df.drop(ds_store_index, inplace=True)
 
+    # recording.conf files
+    recording_conf_index = df[df.file_name == 'recording.conf'].index
+    df.drop(recording_conf_index, inplace=True)
+
+    # chat.txt files
+    chat_txt_index = df[df.file_name == 'chat.txt'].index
+    df.drop(chat_txt_index, inplace=True)
+
+
+def update_skipped_av_files(df: pd.DataFrame) -> pd.DataFrame:
+    """
+    Update the 'file_check' column in the DataFrame based on the 'parent_dir' column.
+
+    Args:
+        df (pd.DataFrame): The DataFrame containing the data.
+
+    Returns:
+        pd.DataFrame: The updated DataFrame.
+    """
+    for idx, row in df.iterrows():
+        if row["parent_dir"] == ("Extraneous files"):
```

Comment:
@dheshanm if `Extraneous files` and `Additional interview files` have sub-directories, files under these sub-directories will be in the `df` framework and have `row["parent_dir"]` as the name of the sub-directory, not `Extraneous files` or `Additional interview files`. Maybe it is safer to use

```py
if "Extraneous files" in row["parent_dir"]:
```

Reply:
I now check if both "Extraneous files" and "Additional interview files" are in `row["parent_dir"]`:

```python
if "Extraneous files" in row["parent_dir"]:
    df.loc[idx, "file_check"] = True
elif "Additional interview files" in row["parent_dir"]:
    df.loc[idx, "file_check"] = True
```